controllers/socket/socket.go

```diff
@@ -96,7 +96,7 @@ func SocketInit(so socketio.Socket) {
 			bytes, _ = tperr.ErrorJSON().Encode()
 			return string(bytes)
 		}
-
+		lob.Save()
```

Comment:
You don't need to save lobby here. What functions lobby.AddPlayer() and RemovePlayer() do is they change the LobbySlot record, which is separate from a Lobby record, and update it. You haven't modified Lobby directly, so you don't need to save it.